utils/Makefile

```diff
@@ -30,6 +32,14 @@ else
 install_files:
 	@echo "Installing Open-CAS utils"
 
+	@install -m 755 -d $(DESTDIR)/etc/opencas
+	@install -m 644 $(UTILS_DIR)/opencas.conf $(DESTDIR)/etc/opencas/opencas.conf
+	@install -m 444 $(UTILS_DIR)/ioclass-config.csv $(DESTDIR)/etc/opencas/ioclass-config.csv
```

Comment:
I know it is just copy-paste, but is there any particular reason why `ioclass-config.csv` is installed with permissions different than the rest of config files? How about changing it to `644`?